docs/content/https/acme.md

```diff
@@ -551,6 +551,68 @@ certificatesResolvers:
 --certificatesresolvers.myresolver.acme.dnschallenge.disablePropagationCheck=true
 ```
 
+#### `propagationRNS`
+
+Use all the recursive nameservers to check the propagation of the TXT records.
+
+It can be used to enforce propagation checks by checking all the recursive nameservers instead of only one.
+
+This is important if you are using `propagationDisableANS`.
+
+```yaml tab="File (YAML)"
+certificatesResolvers:
+  myresolver:
+    acme:
+      # ...
+      dnsChallenge:
+        # ...
+        propagationRNS: true
+```
+
+```toml tab="File (TOML)"
+[certificatesResolvers.myresolver.acme]
+  # ...
+  [certificatesResolvers.myresolver.acme.dnsChallenge]
+    # ...
+    propagationRNS = true
+```
+
+```bash tab="CLI"
+# ...
+--certificatesresolvers.myresolver.acme.dnschallenge.propagationRNS=true
+```
+
+#### `propagationDisableANS`
```

Comment:
Aligning this one with the existing `disablePropagationCheck` option is more consistent.

Reply:
The naming of `disablePropagationCheck` was bad because the main element is "propagation".
Inside lego, `dns.disable-cp` has been deprecated to be replaced by `dns.propagation-disable-ans`.
And the future the options will have this naming with `dns.progation.xxx`.

Reply:
Mmh, considering that, I'm wondering why not introduce the "propagation" configuration node in this PR, hold all propagation options, and deprecate `disablePropagationCheck`.
But if the options remain "ungrouped" in the Traefik configuration, like the current situation, I would prefer the naming as suggested above.

Reply:
Note, `dns.disable-cp` inside lego and `disablePropagationCheck` doesn't have the same meaning:
- `disablePropagationCheck` disables all propagation checks.
- `dns.disable-cp` disables only ANS.

What is your proposal exactly? Should I create a structure or not?

Reply:
Yes, a structure would be better 👍

Reply:
Ok, `delayBeforeCheck` will also be deprecated.